### ISSUE TITLE
Output Activation Layer and Loss Function for training

### ISSUE THREAD

**Issue by IA92** [bacterous/HD-Net_keras#1]
Hi there,
Appreciate your effort in rewriting the code in Keras. I'm trying to use your model for performing prostate segmentation but I realised that you didn't use any activation layer for your final layer, I wonder if you simply forgot it or the model is meant to be trained with linear activation layer and mean-squared error loss function? I just don't understand how to predict 10 classes without the use of softmax activation function. Can you explain a little bit more about the training setup?
Thank you!
